template/DEVELOPING/environment-variables.md

```diff
@@ -19,3 +62,17 @@ environment.
 The reason for `.env*.local` files is so that you can have variables checked
 into version control that can be overridden by `.local` files that are not in
 version control.
+
+In development and testing you probably want to use `.env` files, keeping
```

Comment:
Perhaps add something like

> In general use the `.env` file.  Use the more specific files as needed but sparingly.

Reply:
That's not exactly the point that I was trying to make. I want to recommend using env files for development and testing; but it might make more sense to specify values by other means in production. I'm not sure about recommending `.env` for the general case because I think most variables are going to be environment-specific. I did change the wording slightly to try to clarify what I meant.

Reply:
Ahh -- right.  For what you've set up there's an emphasis toward more specific config files.  In most past JS projects I've dumped lots of config things in the main `.env` file.  What you have seems to make more sense for the TS mindset.